vault/resource_ssh_secret_backend_role.go

```diff
@@ -215,6 +220,10 @@ func sshSecretBackendRoleWrite(d *schema.ResourceData, meta interface{}) error {
 		data["allowed_critical_options"] = v.(string)
 	}
 
+	if v, ok := d.GetOk("allowed_domains_template"); ok {
```

Comment:
For booleans we prefer to use `d.Get` since with the sdk v2 that we are using you cannot reliably determine if a boolean is currently set in the configuration, as opposed to stored in the prior state or set via a default value.

Reply:
+1 on using `d.Get` for boolean field here; we can likely add it to L217 in `data` w/ the rest of the booleans